File: LAB_9_10/LAB10 20161811_20161442/LAB10 20161811_20161442/GameSoftLab10CS/GameSoft/GameSoftRRHH/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GameSoftRRHH
{
    public class Employee
    {
        private int _id;
        private string _name;
        private string _lastName;
        private string _secondLastName;
        private string _DNI;
        private string _gender;
        private string _email;
        private DateTime _birthday;
        private DateTime _hireDate;
        private double _salary;
        private Role _role;

        public int Id { get => _id; set => _id = value; }
        public string Name { get => _name; set => _name = value; }
        public string LastName { get => _lastName; set => _lastName = value; }
        public string SecondLastName { get => _secondLastName; set => _secondLastName = value; }
        public string DNI { get => _DNI; set => _DNI = value; }
        public string Gender { get => _gender; set => _gender = value; }
        public string Email { get => _email; set => _email = value; }
        public DateTime Birthday { get => _birthday; set => _birthday = value; }
        public DateTime HireDate { get => _hireDate; set => _hireDate = value; }
        public double Salary { get => _salary; set => _salary = value; }
        public Role Role { get => _role; set => _role = value; }
    }
}

File: LAB_9_10/20161442_20161811_LAB9_joel/20161442_20161811_LAB9/SERVIDOR/GameSoftRRHH/src/pucp/gamesoft/rrhh/model/Employee.java
package pucp.gamesoft.rrhh.model;
import java.io.Serializable;
import java.util.ArrayList;
import java.util.Date;
public class Employee implements Serializable{

    private int id;
    private ArrayList<Employee> employees;
    private Employee headEmployee;
    private Department department;
    private Role role;
    private String name;
    private String lastName;
    private String secondLastName;
    private String DNI;
    private String email;
    private Date birthday;
    private Date hireDate;
    private double salary;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public ArrayList<Employee> getEmployees() {
        return employees;
    }

    public void setEmployees(ArrayList<Employee> employees) {
        this.employees = employees;
    }

    public Employee getHeadEmployee() {
        return headEmployee;
    }

    public void setHeadEmployee(Employee headEmployee) {
        this.headEmployee = headEmployee;
    }

    public Department getDepartment() {
        return department;
    }

    public void setDepartment(Department department) {
        this.department = department;
    }

    public Role getRole() {
        return role;
    }

    public void setRole(Role role) {
        this.role = role;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getSecondLastName() {
        return secondLastName;
    }

    public void setSecondLastName(String secondLastName) {
        this.secondLastName = secondLastName;
    }

    public String getDNI() {
        return DNI;
    }

    public void setDNI(String DNI) {
        this.DNI = DNI;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public Date getBirthday() {
        return birthday;
    }

    public void setBirthday(Date birthday) {
        this.birthday = birthday;
    }

    public Date getHireDate() {
        return hireDate;
    }

    public void setHireDate(Date hireDate) {
        this.hireDate = hireDate;
    }

    public double getSalary() {
        return salary;
    }

    public void setSalary(double salary) {
        this.salary = salary;
    }
    
    @Override
    public String toString() {
        return DNI + " - " + name + " " + lastName + " " + secondLastName;
    }
    
}